### PR TITLE
bool-styling: remove styling on the pre-generated SVG

### PR DESCRIPTION
**Pull request by david-yz-liu**
@Ian-Stewart-Binks I implemented what we discussed earlier; now there's a flag that determines whether styles are added.

### CODE REVIEW

File: hs/SvgParsing/Generator.hs [hunk 1]
```diff
@@ -163,17 +167,26 @@ convertRectToSVG courseMap rect
...
                          ! A.ry "4"
                          ! A.x (stringValue . show . fst $ shapePos rect)
                          ! A.y (stringValue . show . snd $ shapePos rect)
                          ! A.width (stringValue . show $ shapeWidth rect)
                          ! A.height (stringValue . show $ shapeHeight rect)
-                         ! A.style (stringValue $ "fill:" ++
-                                    getFill (shapeId_ rect) ++
-                                    ";")
-                  concatSVG $ map (convertTextToSVG (shapeType_ rect))
+                         ! if class_ /= "hybrid"
```
Comment (Ian-Stewart-Binks) [inline hs/SvgParsing/Generator.hs]: Why does the code not add styles for hybrids anymore?
black_courseography

**Comment by Ian-Stewart-Binks** [comment]
Hmm. I don't quite remember this, and I can't find where/if I wrote this down. What's the advantage of doing this?

File: hs/SvgParsing/Generator.hs (hunk 1 above)
Comment (david-yz-liu) [inline hs/SvgParsing/Generator.hs]: I don't know what went wrong with your setup; I added a CSS style (line 94-5 of GraphCss.hs) for the hybrids.

**Comment by david-yz-liu** [comment]
One, styling should be done in CSS as much as possible. Two, inline styles prevent CSS styling for things like the "missing" on hover styling for bools. That's what drew my attention to it in the first place.

File: hs/SvgParsing/Generator.hs (hunk 1 above)
Comment (Ian-Stewart-Binks) [inline hs/SvgParsing/Generator.hs]: This is an image generated through the image creation code.

Comment (david-yz-liu) [inline hs/SvgParsing/Generator.hs]: Ah yes, thanks for pointing that out.

**Comment by david-yz-liu** [comment]
Oh, and this came up at the demos as well, with the colours of the nodes overriding a more "faded" look for the focus spotlights.

**Comment by david-yz-liu** [comment]
@Ian-Stewart-Binks Please let me know if you have any concerns about this one, but I'm going ahead and merging this in for tomorrow.

**Comment by Ian-Stewart-Binks** [comment]
@david-yz-liu Looks good, just curious was all :-)

We'll need to change the `1`s to `key`s later.

**Comment by david-yz-liu** [comment]
Thanks, my bad.

**Pull request merged**